src/pages/components/modal/code.mdx

```diff
@@ -58,6 +61,22 @@ documentation, see the Storybooks for each framework below.
 </Column>
 </Row>
 
+### Feature flags
+
+A
+[feature flag](https://carbondesignsystem.com/components/overview/feature-flag/)
+has been added to modal to improve accessibility and changes parts of its
+functionality, not its visual appearance. For more code-specific feature flag
+information, refer to the
+[@carbon/react](https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-modal--overview)
+framework.
+
+The `enable-experimental-focus-wrap-without-sentinels` flag is a new approach to
```

Comment:
The first sentence is kinda run-on. 

```suggestion
The `enable-experimental-focus-wrap-without-sentinels` flag is a new approach to focus wrap behavior, modifying the DOM to no longer include hidden "sentinel nodes" that previously marked the beginning and end of the wrapped focus. This behavior considers all interactive child nodes and wraps focus based on their tab order.
```

Reply:
Hi @thyhmdo! Thanks for the suggestion. Just wanted to confirm—since the same feature flag `enable-experimental-focus-wrap-without-sentinels` exists for Modal, Notification, and Treeview, should I update the copy for those components as well?

Reply:
Yeah, I think so, it'd be great for consistency.